std/assembly/util/number.ts

```diff
@@ -559,16 +559,26 @@ function getCachedPower(minExp: i32): void {
 
 // @ts-ignore: decorator
 @inline
-function grisu2(value: f64, buffer: usize, sign: i32): i32 {
+function grisu2(value: f64, buffer: usize, sign: i32, isSinglePrecision: bool): i32 {
```

Comment:
I prefer to use `isSingle`
```suggestion
function grisu2(value: f64, buffer: usize, sign: i32, isSingle: bool): i32 {
```